sdk/js/src/Client.ts

```diff
@@ -39,24 +42,25 @@ export class DVCClient implements Client {
         this.eventQueue = new EventQueue(environmentKey, this, options?.flushEventsMS)
         this.requestConsolidator = new RequestConsolidator()
         this.eventEmitter = new EventEmitter()
+        this.logger = options?.logger || dvcDefaultLogger({ level: options?.logLevel })
```

Comment:
actually I guess it could be if someone passes null in explicitly while using plain Javascript. Should probably just validate all the constructor properties further up though

Reply:
should we throw an error if options are undefined / null or just log that options are undefined and then create default options?